classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
-[ ] `student` should be in aar_reader.
For ex;-
`attr_accessor :label`
`attr_reader :students`